contracts/src/Teleporter/ITeleporterMessenger.sol

```diff
@@ -62,20 +62,26 @@ interface ITeleporterMessenger {
      * @dev Emitted when Teleporter message is being delivered on destination chain and address,
      * but message execution fails. Failed messages can then be retried.
      */
-    event FailedMessageExecution(
+    event MessageExecutionFailed(
         bytes32 indexed originChainID,
         uint256 indexed messageID,
         TeleporterMessage message
     );
 
     /**
-     * @dev Emitted when a Teleporter message that previously failed to be executed properly
-     * is retried and is successfully executed. The message is then no longer able to be retried again
+     * @dev Emitted when a Teleporter message successfully performs message execution with the
+     * specified destination address and message call data.
+     *
+     * This event is emitted for when messages succeed initial message execution, and if retries
+     * of previously failed message executions now succeed.
+     *
+     * In the successful retry case, the message is then no longer able to be retried again
```

Comment:
```suggestion
     * @dev Emitted when a Teleporter message is successfully executed with the
     * specified destination address and message call data. This can occur either when
     * the message is initially received, or on a retry attempt.
     *
     * Each message received can be executed successfully at most once.
```

(also suggest remove the remaining part of the existing final sentence, github doesn't let me suggest on unchanged lines haha)